src/java.desktop/share/classes/sun/java2d/SunGraphicsEnvironment.java

```diff
@@ -349,53 +350,124 @@ public static GraphicsConfiguration getGraphicsConfigurationAtPoint(
     }
 
     /**
-     * Converts coordinates from the user's space to the device space using
-     * appropriate device transformation.
+     * Returns the bounds of the graphics configuration in device space.
+     *
+     * @param  config the graphics configuration which bounds are requested
+     * @return the bounds of the area covered by this
+     *         {@code GraphicsConfiguration} in device space(pixels).
```

Comment:
```suggestion
     *         {@code GraphicsConfiguration} in device space (pixels).
```
If changed here, all other similar places should be updated too to keep doc comments consistent.

Reply:
I'll fix it and also merge the master, then update the PR.

Reply:
In this case, also consider adding a space between the word and the opening parenthesis in these  _coordinates (x, y)_ and _size (w, h)_ and, probably, a space after comma.

Reply:
Spaces are added.